src/app/common/transactions/bitcoin/use-generate-bitcoin-tx.ts

```diff
@@ -17,22 +18,35 @@ interface GenerateNativeSegwitTxValues {
   amount: Money;
   recipient: string;
 }
-export function useGenerateSignedNativeSegwitTx() {
+export function useGenerateUnsignedNativeSegwitTx() {
   const signer = useCurrentAccountNativeSegwitIndexZeroSigner();
-
   const networkMode = useBitcoinScureLibNetworkConfig();
 
-  return useCallback(
-    (
-      values: GenerateNativeSegwitTxValues,
-      feeRate: number,
-      utxos: UtxoResponseItem[],
-      isSendingMax?: boolean
-    ) => {
-      if (!utxos.length) return;
-      if (!feeRate) return;
+  const { whenWallet } = useWalletType();
+
+  return useMemo(
+    () => ({
+      sign(tx: btc.Transaction) {
```

Comment:
Would it make sense to separate this out into its own hook or rename the file to be more inclusive of both?